assets/css/_comments.scss

```diff
@@ -50,20 +60,28 @@
 .avatar-rounded {
   @include size($rounded-avatar-size);
   border-radius: 50%;
-  margin-right: $tiny-spacing / 2;
+  margin-right: $tiny-spacing;
+  vertical-align: middle;
+}
+
+.avatar-rounded-large {
+  @include size($rounded-avatar-size-large);
+  margin-right: $small-spacing;
 }
 
 .author-information {
   @extend %align-center-flex;
 }
 
 .comment-information {
-  @extend %align-center-flex;
   justify-content: space-between;
   margin-bottom: $tiny-spacing / 2;
+
+  @extend %align-center-flex;
```

Comment:
Is this a thing? I don't see it elsewhere in the code base. If it is a thing, should it be the first declaration?

Reply:
Not sure tbh. I think it's highlighted b/c CSS Comb grabbed it. I'll investigate.